### DISCUSSION TITLE
Would it be possible for ADS to appear in future versions?

### DISCUSSION THREAD

**Discussion by ghost** [Sixze/ALS-Refactored#29] [Ideas]
Would it be possible for ADS to appear in future versions?

https://streamable.com/a6y69q
The video is from this version.
https://github.com/KillerCaveman/ALSv4-4.26-ADS-READY/tree/4.26/AdvancedLocomotionSystemV

**Comment by Sixze** [comment]
It looks like a FPS-specific feature, and ALS is more of a third-person perspective. So, if I decide to add this feature, then most likely this will happen only in the very distant future.

**Reply by ghost** [reply]
I understand, thank you for your reply.

**Reply by xkamronx** [reply]
Here's to hoping we're in the very distant future!

I think adding some sort of camera socket adjustment that would work for ADS and other things and is accessible by BP would be awesome.

As would functions to set up hand IK for things like guns (and any two handed weapon really) easily.
